packages/gatsby-telemetry/src/telemetry.ts

```diff
@@ -336,6 +337,14 @@ export class AnalyticsTracker {
     }
   }
 
+  getIsTTY(): boolean | undefined {
+    try {
```

Comment:
Why would this throw?

Reply:
I'm unsure if there are legit cases where `process.stdout` isn't available. I've tried to build telemetry in a defensive way where it would never cause user-visible errors.

Reply:
Rather that cathing that kind of thing, it's better to add a check. An optional chain is the easiest way here.

Reply:
Refactored